caikit_nlp/modules/text_generation/text_generation_tgis.py

```diff
@@ -203,6 +217,7 @@ def save(self, model_path: str):
                     "sep_token": self._sep_token,
                     "eos_token": self._eos_token,
                     "pad_token": self._pad_token,
+                    # TODO: Add connection_config
```

Comment:
Should the `save()` method not also save the existing connection_config for future use?